index.d.ts

```diff
@@ -2,27 +2,39 @@ interface Options {
   spacer?: number;
   prefix?: string;
   logger?: (method: string, space: string, path: string) => void;
+  color?: boolean;
 }
 
 interface Route {
-  method?: string;
+  method?: string | object;
   path?: string;
 }
 
-interface Stack {
-  route?: {
-    stack: Route[];
-    path?: string;
-  };
-  routerPath?: string;
+interface Layer {
+  name: string;
+  route?: Route;
 }
 
 interface ExpressApp {
-  _router: {
-    stack: Stack[];
+  // Express 3
+  routes?: {
+    get?: Route[][];
+    post?: Route[][];
+    patch?: Route[][];
+    delete?: Route[][];
+  };
+  // Express 4
+  _router?: {
+    stack: Layer[];
+  };
+  // Express 4, 5 Router
+  stack?: Layer[];
+  // Express 5
+  router?: {
+    stack: Layer[];
   };
 }
 
 declare function expressListRoutes(app: ExpressApp, options?: Options): Array<{ method: string; path: string }>;
 
-export { expressListRoutes as default };
+export = expressListRoutes;
```

Comment:
ESM seems incompatible in CommonJS PJs with default export (see https://github.com/microsoft/TypeScript/issues/52086).

Therefore, the existing type definition will result in the following error.

```
This expression is not callable.
  Type 'typeof import("/home/study/workspace/ts-node-oidc/node_modules/express-list-routes/index")' has no call signatures.
```

By setting `export =`, both CommonJS and ES Module can be supported.